kie-wb-tests/kie-wb-tests-rest/src/main/java/org/kie/wb/test/rest/client/WorkbenchClient.java

```diff
@@ -97,6 +117,21 @@
     CreateOrganizationalUnitRequest createOrganizationalUnit(OrganizationalUnit organizationalUnit);
 
     /**
+     * [POST] /organizationalunits
+     */
+    CreateOrganizationalUnitRequest createOrganizationalUnit(String orgUnitName, String owner);
+
+    /**
+     * [POST] /organizationalunits
+     */
+    CreateOrganizationalUnitRequest createOrganizationalUnit(String orgUnitName, String owner, String description);
+
+    /**
+     * [POST] /organizationalunits
+     */
+    CreateOrganizationalUnitRequest createOrganizationalUnit(String orgUnitName, String owner, String description, String groupId);
```

Comment:
It may have a sense.
My intention here was to allow users to use Workbench client without knowledge of its inner parts, without need to instantiate its model.